exporter/dynatraceexporter/README.md

```diff
@@ -256,3 +253,24 @@ Default: `5000`
 ### tags (Deprecated, Optional)
 
 **Deprecated: Please use [default_dimensions](#default_dimensions-optional) instead**
+
+# Limitations
+
+## Cumulative Data Points
+
+When receiving Sum or Histogram metrics with CUMULATIVE temporality, this exporter
+needs to perform CUMULATIVE to DELTA conversion. This conversion can lead to missing
+or inconsistent data, as described below.
+
+### First Data Points are dropped
+
+Due to the conversion the exporter will drop the first received data point,
+as there is no previous data point to compare it to. This can be circumvented by
+exporting DELTA values from the OpenTelemetry SDK.
+
+### Multi-instance deployment
+
+In a multiple-instance deployment of the OpenTelemetry Collector, the conversion
+can produce inconsistent data unless it can be guaranteed that metrics from the
```

Comment:
```suggestion
can produce inconsistent data, unless it can be guaranteed that metrics from the
```

Reply:
Hmm, am I allowed to put a comma there? :sweat_smile:

Reply:
Yeah I guess not, too eager on commas 😅